backend/composer/admin.py

```diff
@@ -194,6 +194,7 @@ class ConnectivityStatementAdmin(
         "sentence__pmcid",
         "knowledge_statement",
     )
+    exclude = ("projection",)
```

Comment:
question (non-blocking): Is there any reason for us to keep projection as a property of the connectivity statement? My understanding is that it was being **replaced** by ProjectionPhenotype

Reply:
follow-up question: If it's meant to replace. How do we map the previous Projection values to the new ProjectionPhenotype?

Reply:
it seems to me that this values hasn't been used so far, looking at the dev and prod data.
Plus the ingestion does not seem is bringing this info in, so we should be fine @afonsobspinto

Reply:
In that case I would suggest to remove the projection attribute from our model. This is a non-blocking suggestion

Reply:
I'll add this as tech debt for the next contract.